packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js

```diff
@@ -22,79 +22,45 @@ const { prefix } = settings;
 /**
  * LightboxMediaViewer Component
  *
- * @param {object} props props object
- * @param {string} props.title LightboxMediaViewer media title
- * @param {string} props.copy LightboxMediaViewer media short description
- * @param {string} props.image LightboxMediaViewer responsive image object
- * @param {boolean} props.open sets whether the modal is open/close
- * @param {Function} props.onClose do something on close in addition, return false to completely replace
+ * @param {object} modalProps props object
+ * @param {string} modalProps.title LightboxMediaViewer media title
+ * @param {string} modalProps.copy LightboxMediaViewer media short description
+ * @param {string} modalProps.image LightboxMediaViewer responsive image object
+ * @param {boolean} modalProps.open sets whether the modal is open/close
  * @returns {*} JSX Object
  */
-const LightboxMediaViewer = ({ title, copy, image, open, onClose }) => {
-  /**
-   * Set default image
-   *
-   * @param {*} image responsive image object
-   * @returns {string} default image url
-   */
-  const defaultImage = image => {
-    return image.uri.md
-      ? image.uri.md
-      : image.uri.lg
-      ? image.uri.lg
-      : image.uri.sm;
-  };
-
-  if (!image.uri.lg) {
-    return null;
-  }
+const LightboxMediaViewer = modalProps => {
```

Comment:
I'm seeing you're still using props below beyond just passing everything into the modal. The way you have it now is you're passing even those props in. I might make sense to break those out and catch everything else...

```javascript
const LightboxMediaViewer = ({ title, image, copy, ...modalProps }) => { ... }
```

Reply:
Okay.